README.md

```diff
@@ -155,12 +156,12 @@ Special thanks to [Jamie Paton](http://jdpaton.github.io) for handing over the `
 
 ### Change Log
 
-This project adheres to [Semantic Versioning](http://semver.org/).  
+This project adheres to [Semantic Versioning](http://semver.org/).
 Every release, along with the migration instructions, is documented on the Github [Releases](https://github.com/rackt/redux/releases) page.
 
 ### Patrons
 
-The work on Redux was [funded by the community](https://www.patreon.com/reactdx).  
+The work on Redux was [funded by the community](https://www.patreon.com/reactdx).
```

Comment:
Please don't remove trailing spaces in README.
Whitespace is significant in Markdown.
These two spaces at the end of the line tell it to break the line.